util/design/README.md

```diff
@@ -163,11 +163,49 @@ TODO
 
 ## LFSR Coefficient Generator Tool
 
-TODO
+The `get-lfsr-coeffs.py` script is used to fetch a list of primitive polynomials for Galois-type LFSRs.
+These coefficients are downloaded from [https://users.ece.cmu.edu/~koopman/lfsr/](https://users.ece.cmu.edu/~koopman/lfsr/).
+When used with the `--fib` switch, the script instead downloads polynomials for Fibonacci-type LFSR.
+The script downloads text files containing the first 100 primitive polynomials for LFSR lengths ranging from 4 to 64 and places them into a temporary folder.
+The script also produces an output file containing a SystemVerilog template with LFSR polynomials for lengths 4 to 64.
+This template contains exactly one polynomial for each LFSR length, which is always the first polynomial listed in the corresponding file.
```

Comment:
The three lines above apply to the Galois variants. 
For the Fibonacci variants, the coefficients are parsed out from a pdf, and the lengths range from 3 to 168.

It probably makes sense to describe these two variants in two paragraphs here.

I also just realized that the script fails with `--fib` since the pdf cannot be downloaded with wget anymore.
This is fixed in #19220.

Reply:
Once list sounds good in principle, but I though the coefficients have to be transformed to work with a specific XOR fan-in architecture?